admin_guide/upgrading_citus.rst

```diff
@@ -63,14 +63,17 @@ Step 1. Update Citus Package
 Step 2. Apply Update in DB
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After installing the new package, run the extension upgrade script:
+After installing the new package and restarting the database, run the extension upgrade script.
 
 .. code-block:: bash
 
+  # you must restart PostgreSQL before running this
   psql -c 'ALTER EXTENSION citus UPDATE;'
 
-  psql -c '\dx'
   # you should see the newer Citus version in the list
+  psql -c '\dx'
+
+During a major version upgrade Citus will refuse to run distributed queries until ALTER EXTENSION is executed.
```

Comment:
I think it should be a note since it does merit attention. Also, its not just until ALTER EXTENSION, but even if you run `yum install` and don't *restart* the DB, we'll error out (albeit on new connections). So we should not that after `yum install` you should (a) restart and (b) alter-extension, and without both Citus will error out.
We can note that this is because object and function definitions are specific to a version, and on version mismatch can cause other issues.